PowerEditor/src/Notepad_plus.cpp

```diff
@@ -5011,7 +5011,12 @@ void Notepad_plus::notifyBufferChanged(Buffer * buffer, int mask)
 
 					// Then we ask user to update
 					if (doReloadOrNot(buffer->getFullPathName(), buffer->isDirty()) != IDYES)
+					{
+						// Since the file has but the user doesn't want to reload it, set state to dirty

```

Comment:
Thanks, I updated the comment! Guess I was not sure whether to say "the file has changed", "the file content has changed" or "the file has changed in the file system" or something else, so I might have decided to just omit the relevant parts of the sentence :)